test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1699,9 +1699,6 @@ class instead of the class it was in when it was queued.
              'enabled': 't', 'Priority': 150}
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='expressq')
 
-        a = {'preempt_sort': 'min_time_since_start'}
```

Comment:
While what you are doing should not affect the test, I'd rather retain it. 
It is better that the tests are explicitly setting it to get the behavior. In future when we have more than one preempt_sort, we will not have to search for tests that expect the value to be min_time_since_start.